Shiyi Li added the 'markdown.qmd' file, Topic Presentation: A Primer of Markdown, to the Introduction section of the Classnote.

Shiyi-Li027 opened the pull request.

jun-yan commented:
+ Watch line width.
+ No manual controlling of spacing.
+ Watch for sectioning levels.

Shiyi-Li027 commented:
I have removed the backslashes, which manually control spacing, revised all the lines' widths to under 80, and removed the heading level setting markdown code.
